digiroad2-viite/src/main/scala/fi/liikennevirasto/viite/ProjectValidator.scala

```diff
@@ -1063,7 +1063,7 @@ class ProjectValidator {
       *
       * @return
       */
-    def checkDiscontinuityOnLastPart: Seq[ValidationErrorDetails] = {
+    def checkDiscontinuityOnLastLinkPart() = {
```

Comment:
Has the return type changed or just removed?

Reply:
I do not remember that I have changed this. Could it be that Idea has removed it with automatic formatting? I think that return type is now assigned from type of returning function.